manifests/cli.pp

```diff
@@ -0,0 +1,92 @@
+# = Pulpcore command line interface
+#
+# This class installs the Pulpcore command line interface (CLI).
+#
+# === Parameters:
+#
+# $pulpcore_url::       URL on which Pulpcore runs
```

Comment:
in other places (`foreman_proxy`, `foreman_proxy_content`) we call this `pulpcore_api_url`.

pulp-cli itself calls it `base_url`.

:bike: shed?

Reply:
Given it's not exposed to users, I don't care that much. Let's start with this.